backend/isic/README.md

```diff
@@ -0,0 +1,18 @@
+To use the module from terminal,run:
+
+```shell script
+$python isic-crawler.py --numjson $json --numimage $img --dermo_filter $bool1 --melanoma_filter $bool2

```

Comment:
Confused command line. what are `$json`, `$img`, ... Don't you have any things to make them differ from `$python`

Reply:
It's just name of the value the users might want to use to set their parameters. Will change now